packages/cerebral-provider-firebase/src/onDisconnect.js

```diff
@@ -1,12 +1,13 @@
 import {
   createRef
 } from './helpers'
+import {FirebaseProviderError} from './errors'
 
 let ref = null
 
 export function setOnDisconnect (path, value) {
   if (ref) {
-    throw new Error('cerebral-provider-firebase: You have already set a disconnect')
+    throw new FirebaseProviderError('You have already a setOnDisconnect')
```

Comment:
shouldn't this be `throw new FirebaseProviderError({ message: 'You have already...')` ? 😄